tools/prepare-machine.ps1

```diff
@@ -235,7 +257,7 @@ if ($Cleanup) {
 } else {
     if ($ForBuild) {
         Download-CoreNet-Deps
-        Download-eBpf-Nuget
+        nuget restore .\xdp.sln
```

Comment:
I think you can just delete this - the build script and CI does a nuget restore already